### DISCUSSION TITLE
Can anyone tell How to install older version of github copilot ?

### DISCUSSION THREAD

**Discussion by abhinav822** [community/community#41959] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Can anyone tell How to install older version of github copilot ?

**Comment by DinhHuynhTienPhu** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/40557558/208227395-6e1023a1-eaed-4aab-8214-7929ff01dbb6.png)
at the extension tab you can click the small down arrow next to uninstall button, then choose install another version

**Reply by sakirtheaa** [reply]
It is not showing for IntelliJ
